### DISCUSSION TITLE
Windows Terminal v1.18.10301.0

### DISCUSSION THREAD

**Discussion by DHowett** [microsoft/terminal#16631] [Announcements]
Hubris! To think that the last 1.18 update would be the _last 1.18 update!_ Alas.

## Bug Fixes

* `wt $COMMAND` no longer blows away the entire process environment, and it now works properly (#16624) 

---

_Binary files inside the unpackaged distribution archive bear the version number `1.18.240130001`._

### Asset Hashes

- Microsoft.WindowsTerminal_1.18.10301.0_8wekyb3d8bbwe.msixbundle_Windows10_PreinstallKit.zip
   - SHA256 `7D2A79C6BED858A0E709E517CC91A241C4BED2755AA54B77EDD129AFB8A85A18`
- Microsoft.WindowsTerminal_1.18.10301.0_8wekyb3d8bbwe.msixbundle
   - SHA256 `DAC95A3D337A315FD1298E2A06F27332F6CADA3E386EA772C179B105A0D817A7`
- Microsoft.WindowsTerminal_1.18.10301.0_arm64.zip
   - SHA256 `9BFBA5883DBA531D785A3BADE526007678C8ACD9FA7AD1B66F8A466B82DAFC82`
- Microsoft.WindowsTerminal_1.18.10301.0_x64.zip
   - SHA256 `38B0E38B545D9C61F1B4214EA3EC6117C0EC348FEB18476D04ECEFB4D7DA723D`
- Microsoft.WindowsTerminal_1.18.10301.0_x86.zip
   - SHA256 `8FB0683AD2A08FED8F76379173FA2157417BEA4633E508B9E6F96E906D0A9D18`

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/terminal/releases/tag/v1.18.10301.0'>Windows Terminal v1.18.10301.0</a>.</em>

**Comment by AnrDaemon** [comment]
> Hubris! To think that the last 1.18 update would be the _last 1.18 update!_ Alas.

Happens every damned time.

**Reply by DHowett** [reply]
I've probably cursed myself into a few more updates today 😆

**Comment by DHowett** [comment]
:mega: This release is being rolled out to Retail/GA today.
